test/libyul/yulOptimizerTests/fullSuite/stack_compressor_msize.yul

```diff
@@ -3,8 +3,8 @@
 
 	function foo_singlereturn_0() -> out
 	{
-		mstore(lt(or(gt(1,or(or(gt(or(or(or(1,gt(or(gt(or(or(keccak256(pc(),or(gt(not(pc()),1),1)),1),not(1)),pc()),1),pc())),lt(or(1,sub(pc(),1)),1)),pc()),1),1),gt(not(pc()),1))),1),1),1)
```

Comment:
Can you use a different instruction than msize here?

Reply:
@chriseth I don't seem to find a nice equivalent to PC() that is not going to be optimized away. Do you have any tips how I could proceed here?

Reply:
calldataload? calldatasize? extcodesize? It has to be non-movable? What about a local function call?